manifests/java.yml

```diff
@@ -126,7 +126,7 @@ tests/:
           Test_XContentSniffing:
             "*": "missing_feature"
             sping-boot-undertow: "bug (APPSEC-11353)"
-            spring-boot: "bug (APPSEC-11353)"
+            spring-boot: v1.20.0
```

Comment:
I think this line should not be touched in this PR, X-Content-Type-Options issues are better addressed on their own.

Reply:
fixed